serverless/README.md

```diff
@@ -0,0 +1,87 @@
+# NFAs Serverless Stack
+
+Boilerplate credit: https://github.com/ixartz/Serverless-Boilerplate-Express-TypeScript[![Twitter](https://img.shields.io/twitter/url/https/twitter.com/cloudposse.svg?style=social&label=Follow%20%40Ixartz)](https://twitter.com/ixartz)
+
+🚀 Serverless Boilerplate is starter code for your backend and REST API based on Serverless framework with Express JS, TypeScript, Prisma and MongoDB. ⚡️ Made with Serverless framework, Live reload, Offline support, ExpressJS, TypeScript, ESLint, Prettier, Husky, Lint-Staged, Jest, Commitlint, Dotenv, VSCode.
+
+Clone this project and use it to create your own backend.
+
+### Features
+
+Developer experience first:
+
+- 🔥 [Serverless framework](https://www.serverless.com)
+- ⚡️ [ExpressJS](http://expressjs.com)
+- ✅ Type checking [TypeScript](https://www.typescriptlang.org) with strict mode
+- 📚 ORM with [Prisma](https://www.prisma.io)
+- 💖 Database with [MongoDB](https://www.mongodb.com/) with in-memory database for local development
+- 📏 Linter with [ESLint](https://eslint.org) with Airbnb configuration
+- 💖 Code Formatter with [Prettier](https://prettier.io)
+- 🦊 Husky for Git Hooks
+- 🚫 Lint-staged for running linters on Git staged files
+- 🚓 Lint git commit with Commitlint
+- 📓 Write standard compliant commit messages with Commitizen
+- 🦺 Unit testing with Jest and Supertest for integration testing
+- 👷 Run tests on pull request with GitHub Actions
+- 🎁 Automatic changelog generation with Semantic Release
+- 💡 Absolute Imports using @ prefix
+- 🗂 VSCode configuration: Debug, Settings, Tasks and extension for ESLint, Prettier, TypeScript, Jest
+- 📖 Local support with Serverless Offline
+- ⚙️ Environment variable with Serverless Dotenv
+- ✨ HTTP Api instead of API gateway for cost optimization
+- 💨 Live reload
+
+### Philosophy
+
+- Minimal code
+- 🚀 Production-ready
+
+### Requirements
+
+- Node.js 14+ and npm
+
+### Getting started
+
+Run the following command on your local environment:
+
+```
+git clone --depth=1 https://github.com/ixartz/Serverless-Boilerplate-Express-TypeScript.git my-project-name
```

Comment:
do we need to include this step?

Reply:
Fixed in [62680f9](https://github.com/fleekxyz/non-fungible-apps/pull/140/commits/62680f92bc6a10e769ee4314edfaafa191ee4194).